Wikipedia/Code/ArticleURLListViewController.swift

```diff
@@ -108,19 +118,6 @@ extension ArticleURLListViewController {
     }
 }
 
-// MARK: - WMFArticlePreviewingActionsDelegate
-extension ArticleURLListViewController {
-    override func shareArticlePreviewActionSelected(withArticleController articleController: WMFLegacyArticleViewController, shareActivityController: UIActivityViewController) {
-        FeedFunnel.shared.logFeedDetailShareTapped(for: feedFunnelContext, index: previewedIndexPath?.item)
```

Comment:
I noticed this line went away in the move - is it needed?

Reply:
Good catch - that is needed! Fixed